modules/op-release-notes-1-13.adoc

```diff
@@ -0,0 +1,141 @@
++
+:FeatureName: using the header and body of the payload in CEL expressions for event matching
+include::snippets/technology-preview.adoc[]
+
+* With this update, when a {pac} pipeline run is triggered by a push event, you can use `/test`, `/test branch:<branchname>`, `/retest`, `/retest branch:<branchname>`, `/cancel`, and `/cancel branch:<branchname>` commands on the corresponding commit comment to re-run or cancel the pipeline run.
+
+* With this update, when using {pac}, you can use remote tasks on remote pipelines. Therefore, you can reuse a complete remote pipeline across multiple repositories. You can override tasks from the remote pipeline by adding a task with the same name.
++
+:FeatureName: using remote tasks on remote pipelines and overriding tasks
+include::snippets/technology-preview.adoc[]
+
+* With this update, when using {pac}, you can view improved information about CI pipeline runs in the Git repository provider. This information includes the namespace and the associated pipeline run.
+
+[id="breaking-changes-1-13_{context}"]
+== Breaking changes
+
+* Before this update, in {pac}, when using a policy group, users who were not a part of policy groups and not allowed explicitly but allowed to run the CI  (via org ownership or otherwise) could sometimes execute pipeline runs by creating events such as pull requests or by entering commands such as `ok_to_test`. With this update, if policy groups are configured, only users that are added to the required policy groups can execute pipeline runs, and users that are a part of the owner organization but not configured in policy groups cannot execute pipeline runs.
+
+[id="known-issues-1-13_{context}"]
+== Known issues
+
+[id="fixed-issues-1-13_{context}"]
+== Fixed issues
+
+* Before this update, some secrets that pipeline runs use for Git authentication could be deleted from the cluster during a cleanup. With this update, a secret is deleted only when all pipeline runs that use it are deleted.
+
+* Before this update, in cases where multiple secrets shared the same prefix and were logged using the git interface, sometimes the concealing process started with a shorter secret, and a part of a longer secret could be displayed in the logs. With this update, when concealing secrets in logs, the process now starts from the longest secret, ensuring that no part of any secret is displayed in the logs.
+
+* Before this update, if you specified a `results` spec for a pipeline, the pipeline run could wrongly fail with a mismatched types error. With this update, if you specify a `results` spec for a pipeline, the results provided by the pipeline are correctly processed.
```

Comment:
@ppitonak You recently tested this I guess. Is this correct?

Reply:
The bug was specifically caused by JSON results, IMHO this description is OK.